platform/stellar/client.go

```diff
@@ -26,17 +27,6 @@ func (c *Client) GetTxsOfAddress(address string) ([]Payment, error) {
 	return payments.Embedded.Records, nil
 }
 
-func (c *Client) GetTxHash(id string) (TxHash, error) {
-	path := fmt.Sprintf("transactions/%s", id)
-
-	var hash TxHash
-	err := c.Get(&hash, path, nil)
-	if err != nil {
-		return hash, err
-	}
-	return hash, nil
-}
-
 func (c *Client) CurrentBlockNumber() (int64, error) {
```

Comment:
make it currentBlockNumber(), because it is not used out of stellar package?

Reply:
I think the goal of the client to expose methods so anyone else can use it